zellij-server/src/panes/grid.rs

```diff
@@ -1766,9 +1766,10 @@ impl Perform for Grid {
                 _ => {}
             }
         } else {
-            let result = debug_log_to_file(format!("Unhandled csi: {}->{:?}", c, params));
-            #[cfg(not(any(feature = "test", test)))]
-            result.unwrap();
+            drop(debug_log_to_file(format!(
```

Comment:
Should I unwrap the result from `debug_log_to_file()`? To me, it looks like an error here should not be the reason a user is unable to zellij.

Reply:
No no, no need to unwrap here at all! If it fails, let it fail.